root.spec

```diff
@@ -146,7 +163,43 @@ done
 export ROOT_INCLUDE_PATH
 
 export ROOTSYS=%i
+
 make INSTALL="$cp" INSTALLDATA="$cp" install
+# Handle debug symbols. We always have a debug subpackage for debug symbols so
+# no point in complicating the standard builds with this.
+
+%if "%{?subpackageDebug:set}" == "set"
+  ELF_DIRS="%i/lib %i/bin"
+  DROP_SYMBOLS_DIRS=""
+
+# optimise the debug symbols, compress them, and split them into separate file
```

Comment:
@davidlt The build worked and the size of the debug rpm was decreased from 123M to 85M

ls -lh */*/*root*
-rw-r--r-- 1 cmsbuild cmsbuild 148M Jul 13 15:00 c51cebe2662f5380aec217041ad1a940/slc6_amd64_gcc491/lcg+root+6.02.10-cms-1-1.slc6_amd64_gcc491.rpm
-rw-r--r-- 1 cmsbuild cmsbuild 158M Jul 13 15:24 da24b007b307dba118e01b65ab5df0f6/slc6_amd64_gcc491/lcg+root+6.02.10-cms2-1-1.slc6_amd64_gcc491.rpm
-rw-r--r-- 1 cmsbuild cmsbuild 123M Jul 13 15:25 da24b007b307dba118e01b65ab5df0f6/slc6_amd64_gcc491/lcg+root-debug+6.02.10-cms2-1-1.slc6_amd64_gcc491.rpm
-rw-r--r-- 1 cmsbuild cmsbuild 158M Jul 15 14:19 fe1fdb9c0bb4a327b3f8d636ab8d83bc/slc6_amd64_gcc491/lcg+root+6.02.10-cms3-1-1.slc6_amd64_gcc491.rpm
-rw-r--r-- 1 cmsbuild cmsbuild  95M Jul 15 14:20 fe1fdb9c0bb4a327b3f8d636ab8d83bc/slc6_amd64_gcc491/lcg+root-debug+6.02.10-cms3-1-1.slc6_amd64_gcc491.rpm